docs/hashes.toml

```diff
@@ -2,7 +2,7 @@
 
 # Spec: ckb
 [ckb]
-spec_hash = "0xc1d2eae53dbb7bedb2ecbb39b1663952226c1a2b147158ce85b1d5780a63bc96"
+spec_hash = "0x3fe86931c2cf5dfa6d7b9447561b5e547937fe4699787bb7ae906218b3f1e6c5"
```

Comment:
The original value in v0.43.0:

https://github.com/nervosnetwork/ckb/blob/bbbe5a58f7024f1157fba75f26ab4e2fdfee1e38/docs/hashes.toml#L3-L5